### DISCUSSION TITLE
OIDCProviderAuthRequestMethod post do not send http headers

### DISCUSSION THREAD

**Discussion by xpoff2007** [OpenIDC/mod_auth_openidc#938] [General]
If OIDCProviderAuthRequestMethod is set to GET the 'Header set directives' in httpd.conf results in headers being set in the 'redirect to IDP 'response, but when OIDCProviderAuthRequestMethod is set to POST NO Headers are set in the 'redirect to IDP'  response. 

1) Is this 'As designed' ? 
2) What are the benefits of POST compared to GET. 
3) Are there any security implications in using GET

**Comment by zandbelt** [comment]
1. not really but it is a side effect of having to "return early" in Apache without getting to a content handler (or redirect)
2. allows for transporting large data values in parameters without hitting URL length limits
3. no, it is the standard
